versioned_docs/version-v0.13.0/int/quickstart/quickstart-cli.md

```diff
@@ -130,13 +130,14 @@ If you have connected all of your charon clients together such that the monitori
 
 This process can take a minimum of 16 hours, with the maximum time to activation being dictated by the length of the activation queue, which can be weeks. You can leave your distributed validator cluster offline until closer to the activation period if you would prefer. You can also use this time to improve and harden your monitoring and alerting for the cluster.
 
-## Step 6. Add the Central Monitoring Token
+## Step 6 - Optional. Add the Monitoring Credentials
 
 :::info
-This step is **optional** but recommended in order for the Obol Team to help with the health of your cluster. 
+This step is **optional** but will help the Obol Team monitor the health of your cluster. It can only be perfomed if the Obol Team has contacted you directly.
```

Comment:
```suggestion
This step is **optional** but will help the Obol Team monitor the health of your cluster. It can only be perfomed if the Obol Team has given you a credential to use.
```